example/zping/main.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
```

Comment:
all the headers should be cleaned up and NetFoundry Inc referenced and <EMAIL ADDRESS> removed

Reply:
This is what is in the existing and other headers in examples

/*
	Copyright 2019 NetFoundry, Inc.

	Licensed under the Apache License, Version 2.0 (the "License");
	you may not use this file except in compliance with the License.
	You may obtain a copy of the License at

	https://www.apache.org/licenses/LICENSE-2.0

	Unless required by applicable law or agreed to in writing, software
	distributed under the License is distributed on an "AS IS" BASIS,
	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
	See the License for the specific language governing permissions and
	limitations under the License.
*/

Reply:
Is that sufficient for each

Reply:
yah that's what we need. It's "NetFoundry Inc." though (no comma)... apparently we need to update more files :/